app/Http/Controllers/Auth/LoginController.php

```diff
@@ -57,6 +57,12 @@ protected function sendLoginResponse(Request $request)
         $request->session()->regenerate();
         $this->clearLoginAttempts($request);
 
+        # Check if it's their first login to reset their password
+        if($user->created_at == $user->updated_at) {
```

Comment:
This will have to end up being changed anyway, just thought that now anyone who has registered after the import will have to change their password which isn't ideal.

Reply:
Leave this bit out for now, lemme think about this one. It might make more sense to add a boolean column of `reset_password`, and check that.